nowcasting_dataset/dataset/split/split.py

```diff
@@ -0,0 +1,80 @@
+""" Function to split datasets up """
+
+import pandas as pd
+from typing import List, Tuple, Union
+
+import logging
+
+from nowcasting_dataset.dataset.split.day import split_day, split_day_random
+from nowcasting_dataset.dataset.split.year import (
+    split_year,
+    TrainValidationTestYear,
+    default_train_test_validation_year,
+)
+from nowcasting_dataset.dataset.split.week import (
+    split_week_random,
+    split_week,
+)
+from enum import Enum
+
+logger = logging.getLogger(__name__)
+
+
+class SplitMethod(Enum):
+    Day = "day"
```

Comment:
I could be wrong but should the elements on the Enum be all-caps?  e.g. replace `Day` with `DAY`?  Because they're basically consts?  But I'm really not sure - I haven't used Python Enums very much!

Reply:
it appears so - https://docs.python.org/3/library/enum.html